src/tests/ftest/container/full_pool_container_create.py

```diff
@@ -63,47 +58,59 @@ def test_no_space_cont_create(self):
                 dkey = (get_random_string(5))
                 akey = (get_random_string(5))
                 try:
-                    dummy_oid = self.cont.write_an_obj(
+                    dummy_oid = cont.write_an_obj(
                         my_str, my_str_sz, dkey, akey, obj_cls="OC_SX")
                     self.d_log.debug("wrote obj {0}, sz {1}".format(write_count,
                                                                     obj_sz))
                     write_count += 1
                 except DaosApiError as excep:
                     if not (err in repr(excep) or err2 in repr(excep)):
-                        self.d_log.error("caught exception while writing "
-                                         "object: {0}".format(repr(excep)))
+                        self.log.error("caught exception while writing "
+                                       "object: {0}".format(repr(excep)))
+                        cont.close()
                         self.fail("caught exception while writing object: {0}"
                                   .format(repr(excep)))
                     else:
-                        self.d_log.debug("pool is too full for {0} byte "
-                                         "objects".format(obj_sz))
+                        self.log.info("pool is too full for {0} byte "
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions